Views/BrowserTab.swift

```diff
@@ -8,10 +8,11 @@
 
 import SwiftUI
 
-struct BrowserTab: View {
+struct BrowserTab: View, Identifiable {
```

Comment:
Does conforming a view struct to identifiable actually do anything? Not sure what exactly you are aiming for, but I have certainly not seen any swiftui code written like this. For example, swiftui view is probably not gonna use this as the view's identity

Please explain, maybe I'll learn something new from you

Reply:
Sorry, it was a left over, removed now.